spikes/SpriteSheet Spike/SpriteSheetManager.h

```diff
@@ -0,0 +1,115 @@
+#pragma once
+#include <SFML/Graphics.hpp>
+#include "Vector2D.h"
+/*
+*@author Aaron Keating
```

Comment:
don't bother with author stuff, commits and github are sufficient for who did what, also prevents team members from editing your code for their needs and instead they'll ask you to edit it which is more work and takes more time